src/groupByKey.ts

```diff
@@ -2,6 +2,17 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
+export function groupByKey<T>(items: T[], key: keyof T): GroupsMap<T> {
   // write code here;
+  return items.reduce((grops, item) => {
```

Comment:
There is a typo in the variable name `grops`. It should be `groups` to match the intended functionality.